pkg/api/file.go

```diff
@@ -179,8 +177,8 @@ func (s *server) fileUploadHandler(w http.ResponseWriter, r *http.Request) {
 		jsonhttp.InternalServerError(w, "metadata marshal error")
 		return
 	}
-	sp = splitter.NewSimpleSplitter(s.Storer, mode)
-	mr, err := file.SplitWriteAll(ctx, sp, bytes.NewReader(metadataBytes), int64(len(metadataBytes)), toEncrypt)
+	pipe = pipeline.NewPipeline(ctx, s.Storer, mode) // TODO just implement `Reset` on the pipeline
```

Comment:
Did we agree to not leave TODO comments in the codebase before? I remember the comment about keeping personal todos somewhere else.